script/testCharacter.py

```diff
@@ -0,0 +1,44 @@
+import Items
+
+class Character:
```

Comment:
Why duplicate Character here? If you want to test Character you should be importing the actual Character class, not creating another one.

Reply:
The issues at hand is the lack of testing that has been done thus far. As it stands, Importing modules outside of a files current directory has not been set up. Character.py does not work in its current state. I am attempting to determine how this needs to be set up. Until then, Character class cannot be imported.

Reply:
Ok, that's fine. If you get the chance to make more changes, add a #TODO comment so we know to migrate this file once testing framework is setting up.